azure-pipelines-release.yml

```diff
@@ -54,11 +70,14 @@ jobs:
       - script: env
         displayName: 'List enviromental variables'
 
-      - script: python -m pip install build
+      - script: python -m pip install cibuildwheel
         displayName: Installing build tools
 
-      - script: python -m build -w .
+      - script: python -m cibuildwheel --output-dir dist/
```

Comment:
From the documentation it seems like `cibuildwheel` also installs the wheels and tests them afterwards. I have jobs further down that do this exact same thing, but they install optional dependencies first. This would effectively mean testing the wheels twice, no? Is there a way to get `cibuildwheel` to do all the testing by itself, with the optional dependencies?

Reply:
> From the documentation it seems like cibuildwheel also installs the wheels and tests them afterwards

Only when CIBW_TEST_COMMAND is defined.